fenix/app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -111,6 +116,7 @@ class HomeMenu(
         label = context.getString(R.string.browser_menu_desktop_site),
         initialState = { context.settings().openNextTabInDesktopMode },
     ) { checked ->
+        context.metrics.track(Event.UsageData.InteractedWithMenu)
```

Comment:
Wondering could we do all these track events in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/home/HomeMenuView.kt#91? We can also add tests for this in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/test/java/org/mozilla/fenix/home/HomeMenuViewTest.kt

Reply:
Moved the tracking event to `HomeMenuView` and fixed the corresponding broken tests.

Working on adding new tests to support telemetry.